lib/env-bundler.js

```diff
@@ -95,6 +101,9 @@ const componentIdExtractors = {
   },
   group: (c) => {
       return c.code;
+  },
+  category: (c) => {
+      return c.code;
   }
 };
 
```

Comment:
Could we reason more in a functional way? Something like: componentIdExtractorFromCode(), componentIdExtractorFromId() and componentIdExtractorFromMetadataName() and then reference them?